paths/reset/fc-commitbroke.md

```diff
@@ -0,0 +1,71 @@
+
+    > If you want to see something kinda cool, open your local repository in a file browser (Finder, My Computer, etc.) and leave it to the side (but in view).
+
+    1. Ensure you are on the correct branch and enter: `git log --oneline`.
+    1. Identify the SHA-1 associated with the **adding file5.md** commit.
+    1. Enter: `git reset --hard SHA-1` where SHA-1 is the SHA-1 associated with the **adding file5.md** commit.
+        If you have your file explorer open, you might have noticed something pretty cool happen!
+    1. Enter: `git log --oneline`, the commit for **adding file6.md** is gone!
+    1. Enter: `ls`. `file6.md` is no longer displayed in the list of files that exist (unlike before).
+
+    ### Wait, I Shouldn't Have Done That!!!
+    OK, so that one rage-induced moment you 'accidentally' deleted that file because you just couldn't stand the sight of it. What if you could bring it back from the dead? You can, with a very nifty command, `reflog`.
+
+     1. Enter: `git reflog`.
+     1. Identify the SHA-1 for the **adding file 6** commit.
+     1. Enter: `git cherry-pick SHA-1`.
+     1. Enter: `git log --oneline`.
+     1. Enter: `git reflog`. Notice any
+
+    You saved the file you deleted! Nice work!
+
+show-me-how:
+tell-me-why: |
+  ## Revert
+  The easiest way to think about `revert` is just making your repository do the exact opposite of an existing commit and creating a new commit to record that change. Revert is useful when trying to reverse the changes made in a specific commit, and even _more_ useful if you pushed a change that your want to reverse to your remote. If you want to reverse a large group of changes and haven't pushed (you can actually use this if you have pushed, but there are some caveats to consider) you should use `reset`. For more information about `reset`, check out the 'Tell me why' section in the [Commit Message Sucks](/on-demand/reset/fc-03) scenario.
+
+  ## Reflog
+  Reflog is a more powerful version of `git log`, it identifies the tip of a branch or other references as they are updated. For instance, when you ran `git log --oneline` and `git reflog` at the end of the exercise, you saw the initial `reset` and the `cherry-pick`, but `git log --oneline` didn't show that information.
+  There are a few things that you should know about `reflog`, such as:
+
+  1. `reflog` is **local** only, so, you other collaborators are not going to be able to find files you deleted in their `reflog`s.
+  1. `reflog` has an expiration date:
```

Comment:
It might be worth noting that this is the default and can be changed in the configs. But then again, it might not. :man_shrugging: